mozjs/src/jsimpls.rs

```diff
@@ -484,9 +484,7 @@ impl JS::ObjectOpResult {
 
 impl Default for ObjectOpResult {
     fn default() -> ObjectOpResult {
-        ObjectOpResult {
-            code_: ObjectOpResult_SpecialCodes::Uninitialized as usize,
-        }
+        ObjectOpResult { code_: ObjectOpResult_SpecialCodes::Uninitialized as usize }
```

Comment:
Were these changes added by mistake?

Reply:
I run `cargo fmt` and I guess this was not formatted properly before.